persist/fs/commitlog/commit_log_test.go

```diff
@@ -368,7 +368,7 @@ func TestCommitLogWrite(t *testing.T) {
 	defer ctx.Close()
 
 	// Call write sync
-	writeCommitLogs(t, scope, stats, ctx, commitLog.Write, writes).Wait()
+	writeCommitLogs(ctx, t, scope, stats, commitLog.Write, writes).Wait()
```

Comment:
yeah don't want to pollute this PR since if I rebase, it'll include other people's changes, i'll resolve the conflicts offline once the PR is approved